ui/enhancement_shaman/sim.ts

```diff
@@ -125,7 +125,8 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecEnhancementShaman, {
 	rotationInputs: ShamanInputs.EnhancementShamanRotationConfig,
 	// Buff and Debuff inputs to include/exclude, overriding the EP-based defaults.
 	includeBuffDebuffInputs: [
-		IconInputs.MP5Buff,
+		IconInputs.BlessingOfWisdom,
+		IconInputs.ManaSpringTotem,
```

Comment:
If I'm understanding the changes correctly, these no longer live in IconInputs right?

Reply:
Yeah I had been planning to rip all of these out later and solely using EP stats unless there's a reason that wouldn't work? I just haven't gotten that far yet

Reply:
I think the reason `includeBuffDebuffInputs` exists is so that sims have the option of limiting their `epStats` to a smaller subset, while still keeping the full set of buff options. For example, computing EPs for Intellect and MP5 can often be pretty unreliable and overly sensitive to inputs for specs like Hunter or Enhancement Shaman, so the sim authors may want to exclude them to speed up EP calculations and reducing user error. But those specs would still want to allow the corresponding buff/debuff options in their sims though.

Reply:
Ah, I see. That's good to know. I can revert the changes then so that those are still used